data/korath/korath weapons.txt

```diff
@@ -53,6 +54,64 @@ effect "grab-strike impact"
 	"velocity scale" -.1
 
 
+outfit "Shunt-Strike Turret"
+	category "Turrets"
+	cost 821000
+	thumbnail "outfit/shunt-strike"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"energy capacity" 260
+	"turret mounts" -1
+	"required crew" 1
+	"asteroid scan power" 16
+	weapon
+		sprite "projectile/shunt-strike"
+			"frame rate" 24
+			"random start frame"
+		"hardpoint sprite" "hardpoint/shunt-strike"
+		"hardpoint offset" 7.
+		sound "shunt-strike"
+		"inaccuracy" 1
+		"turret turn" 2.1
+		"velocity" 10
+		"lifetime" 81
+		"reload" 30
+		"hit effect" "shunt-strike hit"
+		"die effect" "shunt-strike sparkle"
+		"firing energy" 100
+		"firing heat" 150
+		"relative firing shields" 0.001
+		"acceleration" 1
+		"drag" .1
+		"turn" .2
+		"homing" 4
+		"tracking" .9
+		"shield damage" 48
+		"corrosion damage" 3
```

Comment:
On the other hand, it's a rather clumsy weapon, with the some flight characteristics as a grab-strike, not a nice beam like the slicer. I can take another look at it, we did have a few hiccups when converting weapons to the new DoT stuff, but take it for a spin and see how it feels.

We do keep it confined to Korath space and the Wastes, however, as we felt that much of an obvious corrosion effect would detract from the sort of mundane / strange split betwen human space and the rest of the galaxy. Lore wise, you could say it's not used against fragile human hulls for fear of destroying loot.

Reply:
That, and it's notably more effective against Remnant (where they are preferably used) since the Remnant have very good regenerative stats. Based on all the variants we have that run these (including in stage ii), the only variants running them outside of EW space only have a single turret.